src/main/java/easy/mind/com/api/DTO/UserDTO.java

```diff
@@ -30,7 +26,7 @@ public class UserDTO {
     private String password;
 
     @NotNull(message = "email cannot be null")
```

Comment:
As stated previously, NotBlank covers both null and empty constraints. No need for NotNull